google/cloud/aiplatform/training_jobs.py

```diff
@@ -3939,8 +4026,25 @@ def _run(
     ) -> models.Model:
         """Runs the training job and returns a model.
 
-        The training data splits are set by default: Roughly 80% will be used for training,
-        10% for validation, and 10% for test.
+        If training on a Vertex AI dataset, you can use one of the following split configurations:
+            Data fraction splits:
+            Any of ``training_fraction_split``, ``validation_fraction_split`` and
+            ``test_fraction_split`` may optionally be provided, they must sum to up to 1. If
+            the provided ones sum to less than 1, the remainder is assigned to sets as
+            decided by Vertex AI. If none of the fractions are set, by default roughly 80%
+            of data will be used for training, 10% for validation, and 10% for test.
+
+            Predefined splits:
+            Assigns input data to training, validation, and test sets based on the value of a provided key.
+            If using predefined splits, ``predefined_split_column_name`` must be provided.
+            Supported only for tabular Datasets.
```

Comment:
Maybe this should include time-series Datasets.

@thehardikv can you confirm that fraction splits, predefined splits and timestamp splits are all supported by AutoMLForecasting?

Reply:
Confirmed. But we don't really support timestamp splits. Our fraction splits are already time ordered. So internally, timestamp splits are converted into fraction splits. But setting timestamp splits won't throw an error, which I believe  addresses the concern behind the question.

Reply:
Okay, will leave as-is.

Reply:
@thehardikv to confuse users less, I can just always set `timestamp_split_column_name` to None then right?

Reply:
Do you mean as the default value?

Reply:
Yes, basically remove `timestamp_split_column_name` from being set by the user, and internally always pass it as None to the service.

Reply:
👍

Reply:
Yes, I will do that.